templates/blocks/post_meta_fields.twig

```diff
@@ -0,0 +1,45 @@
+{% if (post.post_excerpt) %}
```

Comment:
This template is actually identical to author_meta_fields.twig, except for the variables. Would it make sense here to just have one template with variables? That way we don't have to duplicate all the html and twig logic of escaping the variables.
![Screenshot from 2019-11-26 12-09-02](https://user-images.githubusercontent.com/7604138/69624840-b7c2b480-1045-11ea-9e6f-9362713d7099.png)

Reply:
So you could do something like 
`{% include 'blocks/author_meta_fields.twig' with {description: post.excerpt|striptags|default(site.description)}%}